index.html

```diff
@@ -289,7 +292,40 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-      
+      <form action="https://formspree.io/f/moqypdz" method="POST">
```

Comment:
- [ ] As the previous reviewer pointed out let's follow the given design, It'd be better if you created a CSS rule for the correct font and added it to the form elements, you can see the difference between the fonts in the screenshot:

![image](https://user-images.githubusercontent.com/73140461/142635091-3cb9d7d8-6ebb-4e92-9bef-2175f8f3cd80.png)

Reply:
I added an edit for an optional requirement here, I'm really sorry about missing this at first.